pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -0,0 +1,613 @@
+    rpc Status(StatusRequest) returns (StatusResponse) {}
+
+    // CreatePodSandbox creates a pod-level sandbox. PodSandbox is interpreted 
+    // differently in different runtimes.  For docker, this may mean 
+    // creating the infra container. For hyper, this can create the 
+    // virtual machine.
+    rpc CreatePodSandbox(CreatePodSandboxRequest) returns (CreatePodSandboxResponse) {}
+    // StopPodSandbox stops the sandbox
+    rpc StopPodSandbox(StopPodSandboxRequest) returns (StopPodSandboxResponse) {}
+    // DeletePodSandbox deletes the sandbox.
+    rpc DeletePodSandbox(DeletePodSandboxRequest) returns (DeletePodSandboxResponse) {}
+    // PodSandboxStatus returns the Status of the PodSandbox.
+    rpc PodSandboxStatus(PodSandboxStatusRequest) returns (PodSandboxStatusResponse) {}
+    // ListPodSandbox returns a list of SandBox.
+    rpc ListPodSandbox(ListPodSandboxRequest) returns (ListPodSandboxResponse) {}
+    // PortForward copies the data from the port in the PodSandBox to the stream.
+    rpc PortForward(stream PortForwardRequest) returns (stream PortForwardResponse) {}
+
+    // CreateContainer creates a new container in specified PodSandbox
+    rpc CreateContainer(CreateContainerRequest) returns (CreateContainerResponse) {}
+    // StartContainer starts the container.
+    rpc StartContainer(StartContainerRequest) returns (StartContainerResponse) {}
+    // StopContainer stops a running container with a grace period (i.e., timeout).
+    rpc StopContainer(StopContainerRequest) returns (StopContainerResponse) {}
+    // RemoveContainer removes the container.
+    rpc RemoveContainer(RemoveContainerRequest) returns (RemoveContainerResponse) {}
+    // ListContainers lists all containers by filters.
+    rpc ListContainers(ListContainersRequest) returns (ListContainersResponse) {}
+    // ContainerStatus returns the container status.
+    rpc ContainerStatus(ContainerStatusRequest) returns (ContainerStatusResponse) {}
+    // Exec execute a command in the container.
+    rpc Exec(stream ExecRequest) returns (stream ExecResponse) {}
+}
+
+// Image service defines the public APIs for managing images
+service ImageService {
+    // ListImages lists existing images.
+    rpc ListImages(ListImagesRequest) returns (ListImagesResponse) {}
+    // ImageStatus returns the status of the image.
+    rpc ImageStatus(ImageStatusRequest) returns (ImageStatusResponse) {}
+    // PullImage pulls a image with authentication config.
+    rpc PullImage(PullImageRequest) returns (PullImageResponse) {}
+    // RemoveImage removes the image.
+    rpc RemoveImage(RemoveImageRequest) returns (RemoveImageResponse) {}
+}
+
+message VersionRequest {
+    // The version of the gRPC API
+    required string version = 1 [default="0.1.0"];
+}
+
+message VersionResponse {
+    // The version of the gRPC API
+    required string version = 1 [default="0.1.0"];
+    // The name of the container runtime.
+    required string containerRuntimeName = 2;
```

Comment:
Just a small nit :-). Field names in messages should be in snake_case rather than camelCase, per the style guide: https://developers.google.com/protocol-buffers/docs/style#message-and-field-names. I noticed all of the message field names in this file are in camelCase.